docs/reference/release-notes/highlights.asciidoc

```diff
@@ -1,6 +1,5 @@
-[chapter]
 [[release-highlights]]
-= What's new in {minor-version}
+== What's new in {minor-version}
```

Comment:
yeah, this was created by automation.  If if fix it now, it will come back the next time some generates the release notes. I will add a comment to this effect to this file too and will address in a follow up.

Reply:
thanks!